# Aufgabe

Eine Wasserleitung CU 35$\times$1.2 soll einen Volumenstrom von $\dot V = 4.0\,\frac{\mathrm{m}^3}{\mathrm{h}}$ transportieren. 

Zulässig ist eine Geschwindigkeit von $v = 1.0\,\frac{\mathrm{m}}{\mathrm{s}}$, bei höheren Geschwindigkeiten können Strömungsgeräusche auftreten.

1. Berechnen Sie die Geschwindigkeit, die sich in dem Rohr einstellt.
2. Welcher Durchmesser ist mindestens zu wählen, um die zugelassene Geschwindigkeit nicht zu überschreiten.

Rechnen Sie schrittweise und kommentieren Sie Ihre Lösung ausführlich.

Die Zusammenhänge zwischen Volumenstrom, Durchmesser und Geschwindigkeit sind durch die folgenden Formeln gegeben:

\begin{align}
  \dot V &= A\,v,
  &
  A &= \dfrac{\pi\,d^2}{4}
  \\
  v &= \dfrac{\dot V}{A}
  \\
  A &= \dfrac{\dot V}{v},
  &
  d &= \sqrt{\dfrac{4\,A}{\pi}}
\end{align}

Das Modul `math` sollten Sie importieren. Das ist in der folgenden Zelle voreingestellt.

Sie finden die Zahl $\pi$ als `math.pi`.

Die Wurzel einer Zahl `x` können Sie als `math.sqrt(x)` berechnen.

In [7]:
import math

1. Berechnen Sie die Geschwindigkeit, die sich in dem Rohr einstellt.

In [8]:
# Ihre Lösung beginnt hier
d = (35-2*1.2)*1e-3 # lichter Durchmesser in m

A = math.pi*d**2/4 
A # lichter Querschnitt in m**2

0.0008346897521322724

In [9]:
dV_h = 4 # Volumenstrom in m**3/h

v = dV_h/(3600*A)
v # Geschwindigkeit in m/s

1.3311665900685872

Diese Geschwindigkeit ist zu hoch! Es muss ein größerer Querschnitt gewählt werden.

2. Welcher Durchmesser ist mindestens zu wählen, um die zugelassene Geschwindigkeit nicht zu überschreiten.

In [10]:
v = 1 # Geschwindigkeit in m/s

A = dV_h/(3600*v) # Querschnitt in m**2

d = math.sqrt(4*A/math.pi)*1000
d # Durchmesser in mm

37.61263890318375

Es wird mindestens ein Durchmesser von $37.6\,mm$ benötigt. Laut Tabellenbuch muss ein Querschnitt CU 42$\times$1.5 gewählt werden. Die Geschwindigkeit ist dann:

In [11]:
d = (42-2*1.5)*1e-3

A = math.pi*d**2/4 
A # lichter Querschnitt in m**2

0.001194590606527519

In [12]:
dV_h = 4 # Volumenstrom in m**3/h

v = dV_h/(3600*A)
v # Geschwindigkeit in m/s

0.9301187411316844

Die Geschwindigkeit ist nun mit $v = 0.93\,\frac{m}{s}$ zulässig.